docs/references/virtual_environments_markdown.md

```diff
@@ -0,0 +1,47 @@
+Virtual Environments
+====================
+
+Developing in a Virtual Environment
+-----------------------------------
+
+We recommend that you develop your custom plugins inside a virtual
+environment. This creates an isolated development environment that
+minimizes the risk of creating a package that Alteryx Designer and other
+users won\'t be able to use. It does this by keeping the dependencies
+required by different projects separate.
```

Comment:
TW review:

We recommend that you develop your custom plugins inside a virtual
environment. This creates an isolated development environment that
minimizes the risk of creating a package that Alteryx Designer and other
users won't be able to use. To do this, it keeps the dependencies
required by different projects separate.

Reply:
fixed